include/deal.II/distributed/shared_tria.h

```diff
@@ -228,11 +228,24 @@ namespace parallel
       /**
        * Constructor.
        *
-       * If @p allow_aritifical_cells is true, this class will behave similar
-       * to parallel::distributed::Triangulation in that there will be locally
-       * owned, ghost and artificial cells.
+       * The flag @p allow_artificial_cells can be used to enable artifical
+       * cells. If enabled, this class will behave similarly
+       * to parallel::distributed::Triangulation and
+       * parallel::fullydistributed::Triangulation in the sense that there will
+       * be locally-owned cells, a single layer of ghost cells, and
```

Comment:
As far as I can tell we write this without an hyphen

```suggestion
       * be locally owned cells, a single layer of ghost cells, and
```